README.md

```diff
@@ -27,6 +27,10 @@ envd (`ɪnˈvdɪ`) is a **machine learning development environment** for data sc
 
 🔁 **Repeatable builds & reproducible results** - You can reproduce the same dev environment on your laptop, public cloud VMs, or Docker containers, without any change in setup.
 
+<p align="center">
+  <img src="docs/images/demo.gif" width="600"/>
```

Comment:
This gif is huge. Can we upload it to discussion or issue or gist?

Reply:
SGTM.

Reply:
/hold

Reply:
/hold-cancel

Reply:
Can delete the gif and squash